pkg/daemon/writer.go

```diff
@@ -19,55 +27,106 @@ const (
 	machineConfigDaemonSSHAccessAnnotationKey = "machineconfiguration.openshift.io/ssh"
 	// MachineConfigDaemonSSHAccessValue is the annotation value applied when ssh access is detected
 	machineConfigDaemonSSHAccessValue = "accessed"
+
+	nodeWriterKubeconfigPath = "/var/lib/kubelet/kubeconfig"
```

Comment:
I like the idea of using kubelet's kubeconfig just for node writes as we do here. The MCD can still have node list/get/watch perms used everywhere else

Reply:
Sure, mostly had a question on https://github.com/openshift/machine-config-operator/pull/3143/files#r870410935

I'd also like to try to rebase my PR on this, so was wondering which of the two directions you were thinking of taking